apps/backend/user/src/login.ts

```diff
@@ -0,0 +1,57 @@
+import { Request, Response } from "express";
+import { IDatabase } from "pg-promise";
+import { User } from "./models/user";
+import bcrypt from "bcryptjs";
+
+const login = async (req: Request, res: Response, db: IDatabase<object>) => {
+  const { email, password } = req.body;
+
+  if (!email || !password) {
+    return res.status(400).json({ error: "Email and password are required" });
+  }
+
+  try {
+    const user = await db.oneOrNone<User>(
+      "SELECT * FROM users WHERE email = $1",
+      [email],
+    );
+
+    if (!user) {
+      return res.status(401).json({ error: "Invalid email" });
```

Comment:
resops (Max) might not like this because it allows a malicious agent to sniff out which emails have an account on the site, making it easier to target accounts for cracking their passwords etc

Reply:
I'll change both catches to just return "invalid email or password"